Decompiler/PscDecompiler.cpp

```diff
@@ -1,18 +1,13 @@
 #include "PscDecompiler.hpp"
 
-#include <map>
 #include <cassert>
 #include <cstdint>
 #include <iostream>
-#include <sstream>
```

Comment:
The `sstream` include removal here, and possible a few of the other includes that were removed were only needed for the `TRACE_DECOMPILATION` code, so should be added back at the same time.